docker/1.6.0/py3/Dockerfile.cpu

```diff
@@ -101,7 +101,7 @@ RUN ${PIP} install --no-cache --upgrade \
     onnx==1.6.0 \
     numpy==1.17.2 \
     pandas==0.25.1 \
-    Pillow==6.2.0 \
+    Pillow==7.1.0 \
```

Comment:
why 6.2.2 in py 2 and 7.1.0 in py 3 ? Any specific reason ?

Reply:
https://pillow.readthedocs.io/en/stable/releasenotes/7.0.0.html#python-2-7